internal/clients/project.go

```diff
@@ -29,3 +30,22 @@ func GetParentOrganizationIDByProjectID(ctx context.Context, client *Client, pro
 
 	return project.Parent.ID, nil
 }
+
+func CreateProject(ctx context.Context, client *Client, name, organizationID string) (string, error) {
+	projectOrg := &resourcemodels.HashicorpCloudResourcemanagerResourceID{
+		ID:   organizationID,
+		Type: resourcemodels.NewHashicorpCloudResourcemanagerResourceIDResourceType("ORGANIZATION"),
+	}
+	projectParams := project_service.NewProjectServiceCreateParamsWithContext(ctx)
+	projectParams.Body = &resourcemodels.HashicorpCloudResourcemanagerProjectCreateRequest{
+		Name:   name,
+		Parent: projectOrg,
+	}
+
+	createProjectResp, err := client.Project.ProjectServiceCreate(projectParams, nil)
+	if err != nil {
+		return "", fmt.Errorf("unable to create project '%s' with organization ID %s: %v", name, organizationID, err)
+	}
+
+	return createProjectResp.Payload.Project.ID, nil
```

Comment:
I think you'll want to return the whole Project object
```suggestion
	return createProjectResp.Payload.Project, nil
```